services/src/security_mode.cpp

```diff
@@ -108,6 +187,14 @@ int security_mode_check_request(security_mode_transport transport, uint16_t id)
         }
     }
 
+    if (transport == SECURITY_MODE_TRANSPORT_USB) {
+        switch (id) {
+            case CTRL_REQUEST_SET_PROTECTED_STATE: {
```

Comment:
Let's allow CTRL_REQUEST_SET_PROTECTED_STATE under BLE as well so a mobile app can be used to unprotect a device.